bootstrap.sh

```diff
@@ -8,6 +8,9 @@ export TEMP_DIR=/tmp
 export TMPDIR=/tmp
 export TMP_DIR=/tmp
 
+# install expect
+apt-get install -y expect
```

Comment:
Is there a reason for the `update`? Local tests don't require that. Also if I do I will also have to do `autoremove`